airbyte-integrations/connectors/source-unleash/source_unleash/unleash.yaml

```diff
@@ -0,0 +1,55 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  selector_features:
+    extractor:
+      field_pointer: ["features"]
+  requester:
+    url_base: "{{ config['api_url'] }}/api/client"
+    http_method: "GET"
+    request_options_provider:
+      request_headers:
+        "Content-Type": "application/json"
+        "Authorization": "{{ config['api_token'] }}"
+
+    error_handler:
+      type: "DefaultErrorHandler"
+      error_handlers:
+        - response_filters:
+            - http_codes: [401]
+              action: FAIL
+            - http_codes: [403]
+              action: RETRY
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  # API Docs: https://docs.getunleash.io/reference/api/legacy/unleash/client/features
+  features_stream:
+    $ref: "*ref(definitions.base_stream)" 
+    retriever:
+      $ref: "*ref(definitions.retriever)" 
+      record_selector:
+          $ref: "*ref(definitions.selector_features)"      
+    $options:
+      name: "features"
+      primary_key: "name"
+      path: "/features?{{ ('project=' ~ config['project_name']) if config['project_name'] is defined else '' }}&{{ ('namePrefix=' ~ config['nameprefix']) if config['nameprefix'] is defined else '' }}"
+
+streams:
+  - "*ref(definitions.features_stream)"
```

Comment:
Are you planning to only ingest this stream? Or to add others like https://docs.getunleash.io/reference/api/unleash/get-users ?

Reply:
we only have a clear use case for the client side API as of now (we are not using the admin API) and `features` is the only GET endpoint on this API. But happy to extend it as a future iteration if some users need other streams 👍